config/routes.rb

```diff
@@ -5,7 +5,11 @@
         get 'find', to: 'finder#show'
         get 'find_all', to: 'finder#index'
         get 'random', to: 'random#show'
-        get 'transactions', to: 'transactions#index'
+        get ':id/transactions', to: 'transactions#index'
```

Comment:
I think I figured out how to get this nested under invoice_items but still have the controller be where we want it. -note for when we refactor